Unable to build neovim - payment required for libtermkey download

### Problem

I am trying to build neovim, and get this:

```
[  7%] Performing download step (download, verify and extract) for 'libtermkey'
-- verifying file...
       file='/xxxxx/.home/linux/aj_sources/neovim/.deps/build/downloads/libtermkey/libtermkey_0.22.orig.tar.gz'
-- SHA256 hash of
    /xxxxx/.home/linux/aj_sources/neovim/.deps/build/downloads/libtermkey/libtermkey_0.22.orig.tar.gz
  does not match expected value
    expected: '6945bd3c4aaa83da83d80a045c5563da4edd7d0374c62c0d35aec09eb3014600'
      actual: 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
-- File already exists but hash mismatch. Removing...
-- Downloading...
   dst='/xxxxx/.home/linux/aj_sources/neovim/.deps/build/downloads/libtermkey/libtermkey_0.22.orig.tar.gz'
   timeout='none'
   inactivity timeout='none'
-- Using src='https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/libtermkey/0.22-1/libtermkey_0.22.orig.tar.gz'
CMake Error at libtermkey-stamp/download-libtermkey.cmake:170 (message):
  Each download failed!

    error: downloading 'https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/libtermkey/0.22-1/libtermkey_0.22.orig.tar.gz' failed
          status_code: 22
          status_string: "HTTP response code said error"
          log:
          --- LOG BEGIN ---
          Uses proxy env variable https_proxy == 'localhost:3122'
    Trying 127.0.0.1:3122...

  Connected to (nil) (127.0.0.1) port 3122 (#0)

  allocate connect buffer

  Establish HTTP proxy tunnel to launchpad.net:443

  CONNECT launchpad.net:443 HTTP/1.1

  Host: launchpad.net:443

  User-Agent: curl/7.86.0

  Proxy-Connection: Keep-Alive



  HTTP/1.1 402 Payment Required

  Cache-Control: no-cache

  Pragma: no-cache

  Content-Type: text/html; charset=utf-8

  Proxy-Connection: Keep-Alive

  Connection: Keep-Alive

  Content-Length: 1268



  The requested URL returned error: 402

  CONNECT phase completed

  Closing connection 0



          --- LOG END ---




make[3]: *** [build/src/libtermkey-stamp/libtermkey-download] Error 1
make[3]: Leaving directory `/xxxxx/.home/linux/aj_sources/neovim/.deps'
make[2]: *** [CMakeFiles/libtermkey.dir/all] Error 2
make[2]: Leaving directory `/xxxxx/.home/linux/aj_sources/neovim/.deps'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/xxxxx/.home/linux/aj_sources/neovim/.deps'
make: *** [deps] Error 2
```

The above happens when I am trying to build the latest version of neovim on my corporate server, at work. And it wasn't like that before. I tried to build neovim at home, and this problem is not happening.

The above suggests that launchpad.net may have recently introduced so sort of IP checks and requires the commerical entities to pay? While I may understand the reasons behind such decision, this adds the whole new level of complexity to the building process. And, while the amount of payment may be reasonable for my company, the amount of paperwork to make it happen may be prohibitive...

So, the question is, are there any alternatives to launchpad, to be used for neovim building?

### Steps to reproduce

make distclean
CC=/path/to/bin/cc make CMAKE_BUILD_TYPE=Release CMAKE_INSTALL_PREFIX=/xxxxxx/linux/local


### Expected behavior

Neovim build process downloads all the required dependencies.

### Neovim version (nvim -v)

Commit c14aa66cc

### Vim (not Nvim) behaves the same?

not relevant

### Operating system/version

Red Hat Enterprise Linux Server release 7.9

### Terminal name/version

tmux

### $TERM environment variable

tmux-256color

### Installation

build from repo

--- a/cmake.deps/deps.txt
+++ b/cmake.deps/deps.txt
@@ -16,16 +16,16 @@ LUAROCKS_SHA256 a0b36cd68586cd79966d0106bb2e5a4f5523327867995fd66bee4237062b3e3b
 UNIBILIUM_URL https://github.com/neovim/unibilium/archive/d72c3598e7ac5d1ebf86ee268b8b4ed95c0fa628.tar.gz
 UNIBILIUM_SHA256 9c4747c862ab5e3076dcf8fa8f0ea7a6b50f20ec5905618b9536655596797487
 
-LIBTERMKEY_URL https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/libtermkey/0.22-1/libtermkey_0.22.orig.tar.gz
+LIBTERMKEY_URL https://github.com/neovim/deps/raw/master/opt/libtermkey-0.22.tar.gz
 LIBTERMKEY_SHA256 6945bd3c4aaa83da83d80a045c5563da4edd7d0374c62c0d35aec09eb3014600
 
-LIBVTERM_URL https://launchpad.net/libvterm/trunk/v0.3/+download/libvterm-0.3.1.tar.gz
+LIBVTERM_URL https://github.com/neovim/deps/raw/master/opt/libvterm-0.3.1.tar.gz
 LIBVTERM_SHA256 25a8ad9c15485368dfd0a8a9dca1aec8fea5c27da3fa74ec518d5d3787f0c397
 
 LUV_URL https://github.com/luvit/luv/archive/093a977b82077591baefe1e880d37dfa2730bd54.tar.gz
 LUV_SHA256 222b38b6425f0926218e14e7da81481fdde6f9660c1feac25a53e6fb52e886e6
 
-LPEG_URL http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-1.0.2.tar.gz
+LPEG_URL https://github.com/neovim/deps/raw/master/opt/lpeg-1.0.2.tar.gz
 LPEG_SHA256 48d66576051b6c78388faad09b70493093264588fcd0f258ddaab1cdd4a15ffe
 
 LUA_COMPAT53_URL https://github.com/keplerproject/lua-compat-5.3/archive/v0.9.tar.gz

